src/toasts/toasts.stories.js

```diff
@@ -0,0 +1,21 @@
+import { storiesOf } from "@storybook/html";
+import html from "./toasts.stories.html";
+import "./toasts.css";
+import ejs from "ejs";
+
+import { htmlStory } from "../story-helpers";
+import { showToast } from "./toasts.js";
+// storiesOf("OpenMRS Styleguide", module).add(
+//   "Toasts",
+//   () => htmlStory(html)
+// );
+
+storiesOf("OpenMRS Styleguide", module).add("MyState", () => {
+  function init() {
+    var button = document.querySelector(".js-toggle");
+    button.addEventListener("click", showToast("This is a toast"));
+  }
+
+  document.addEventListener("DOMContentLoaded", init);
+  return '<button class="js-toggle">Toggle</button>';
```

Comment:
Please put the html into a separate file. See [here](https://github.com/openmrs/openmrs-esm-styleguide/blob/master/src/cards/cards.stories.js#L10) and [here](https://github.com/openmrs/openmrs-esm-styleguide/blob/master/src/cards/cards.stories.html) for example

Reply:
I will modify that after I will figure out how to create a story for an react component or dom or what is this.

Reply:
Can you help me with stylling? Cant center that X icon in the button and can't resize it.